elements/pfe-popover/src/pfe-popover.scss

```diff
@@ -0,0 +1,223 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+@import "../../pfe-absolute-position/src/pfe-absolute-position.scss";
+
+$LOCAL: popover;
+
+// Example local variable, using BEM format
+// --pfe-popover__region--Property: value;
+
+// This variable is global so that helper functions can reference it
+$LOCAL-VARIABLES: (
+  Fonsize: 16px,
+  MinWidth: 360px,
+  MaxWidth: 360px,
+  BoxShadow: 0px 7px 15px #00000026,
+  Position: relative,
+  BackgroundColor: #{$pf-color-white},
+  Color: #{$pf-color-black-900},
+  content: (
+    Padding: #{$pf-spacer-sm},
+  ),
+  header: (
+    FontWeight: 500,
+    MarginBottom: #{$pf-spacer-xs},
+  ),
+  footer: (
+    MarginTop: #{$pf-spacer}
+  ),
+  closeButton: (
+    Top: #{$pf-spacer-sm},
+    Right: #{$pf-spacer-sm},
+    Position: absolute,
+    Color: #6B6E74,
+  ),
+  closeButton--hover: (
+    Color: #151515,
+  ),
+  arrow: (
+    Width: 30px,
+    Height: 30px,
+    Position: absolute,
+    BackgroundColor: #{$pf-color-white},
+  ),
+  arrow--top: (
```

Comment:
#sand  do these arrow properties need to be CSS variables? I'm trying to think of a use case where they would want to manually move the arrow with CSS instead of attributes